tests/data/CMakeLists.txt

```diff
@@ -98,3 +99,8 @@ add_custom_command(
   DEPENDS ${BTF_DATA_HEX} ${FUNC_LIST_HEX} ${CONFIGURE_BTF_HEADERS})
 
 add_custom_target(debuginfo_btf_data DEPENDS ${BTF_DATA_H})
+
+# BTF doesn't support C++, so we only generate a data_source_cxx executable
+# to run the semantic_analyser tests on.
```

Comment:
Nit: we're not running any semantic analyser tests, rather field analyser tests:
```suggestion
# to run the field_analyser tests on.
```